app/connectors/IFConnector.scala

```diff
@@ -29,14 +29,15 @@ trait IFConnector {
   val appConfig: AppConfig
 
   val headerCarrierConfig: Config = HeaderCarrier.Config.fromConfig(ConfigFactory.load())
+  val PostCalculateIncomeTaxLiability = "1897"
```

Comment:
this is not the right place to have API specific reference under the trait which is a for a general use